evmrpc/config.go

```diff
@@ -9,6 +9,16 @@ import (
 	"github.com/spf13/cast"
 )
 
+// calculateDefaultWorkerPoolSize returns the default worker pool size.
+// Capped at 64 to prevent excessive goroutines on high-core machines.
+func calculateDefaultWorkerPoolSize() int {
+	workers := runtime.NumCPU() * 2
+	if workers > 64 {
+		return 64
+	}
+	return workers
```

Comment:
You can write this entire body as:
```go
min(64, runtime.NumCPU() * 2)
```

..at which point, i would recommend against refactoring it into its own function and moving the godoc to be an inline comment that is also reflected in config documentation.